journal.py

```diff
@@ -17,10 +17,8 @@
 
 from peewee import *
 
-
-# TODO: make the database point to a seperate home directory for the specific OS
-db = SqliteDatabase('diary.db')
-
+path = os.environ['HOME'] + '/.tnote'
```

Comment:
But I think this would break when running in a **Windows** environment don't you. Something like 

```python
HOME =expanduser("~")
db_name = 'diary.db'
db_location = os.path.join(HOME, db_name)
SCRIPT_DIRECTORY = os.path.dirname(os.path.abspath(__file__)) 
```
I suggest you test on it. Will merge it then